src/entity/posts/utils.ts

```diff
@@ -227,6 +230,11 @@ export const createExternalLink = async (
   });
 };
 
+export const generateTitleHtml = (
+  title: string,
+  mentions: MentionedUser[],
+): string => `<p>${renderMentions(title, mentions)}</p>`;
```

Comment:
We have to be careful here because this is gonna be in a `innerHtml`.
We have to make sure it's a safe string with no html tags or javascript snippets, or in other words, we have to escape the html. You can have a look at the markdown renderer to see how we escape it.
You need to apply it on the title. Please also add some tests to check that